libs/networks.py

```diff
@@ -71,11 +71,13 @@ def __init__(self, name="u2net"):
         else:
             raise Exception("Unknown u2net model!")
         try:
+            project_dir = os.path.join(os.path.dirname(__file__), "..")
```

Comment:
```suggestion
            project_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
```
I suppose this would be more robust